source/adapters/cuda/enqueue.cpp

```diff
@@ -187,6 +187,146 @@ bool hasExceededMaxRegistersPerBlock(ur_device_handle_t Device,
   return BlockSize * Kernel->getRegsPerThread() > Device->getMaxRegsPerBlock();
 }
 
+// Helper to compute and put in good shape kernel parameters from workload
```

Comment:
What do you mean by "put in good shape"? That give the reader far too much latitude for imagination

Reply:
I've just removed this wording from the comment as it didn't really add anything except potential confusion in this case.